pyproject.toml

```diff
@@ -83,12 +83,32 @@ source = 'https://github.com/mscheltienne/antio'
 tracker = 'https://github.com/mscheltienne/antio/issues'
 
 [tool.cibuildwheel]
-build-frontend = "build[uv]"
-test-command = "pytest tests --cov=src/antio --cov-report=xml --cov-config=pyproject.toml"
-test-extras = ["mne", "test"]
+build = "cp312-*"
+skip = "*musllinux*"
+archs = "native"
 
 [tool.cibuildwheel.linux]
-before-all = "sudo yum update -y & sudo yum install https://repo.ius.io/ius-release-el$(rpm -E '%{rhel}').rpm & sudo yum update -y & yum install -y python3-devel"
+repair-wheel-command = [
+  "auditwheel repair -w {dest_dir} {wheel}",
+  "pipx run abi3audit --strict --report {wheel}",
+]
+[tool.cibuildwheel.macos]
+repair-wheel-command = [
+  "delocate-wheel --require-archs {delocate_archs} -w {dest_dir} -v {wheel}",
+  "pipx run abi3audit --strict --report {wheel}",
+]
+
+[tool.cibuildwheel.windows]
+before-build = "pip install delvewheel"
+repair-wheel-command = [
+  "delvewheel repair -w {dest_dir} {wheel}",
+  "pipx run abi3audit --strict --report {wheel}",
+]
+
+[[tool.cibuildwheel.overrides]]
+select = "*aarch64*"
```

Comment:
I actually don't like this so much. There isn't really any harm in running the tests in `cibuildwheel` even if we do then test elsewhere / in different jobs. Moreover, having this select here makes it so that `python -m cibuildwheel` locally *will not* run tests, which is a bummer.
```suggestion
```